src/Microsoft.Health.SqlServer/Features/Schema/SchemaInitializer.cs

```diff
@@ -87,38 +89,32 @@ public async Task InitializeAsync(bool forceIncrementalSchemaUpgrade = false, Ca
                     int current = _schemaInformation.Current ?? 0;
                     for (int i = current + 1; i <= _schemaInformation.MaximumSupportedVersion; i++)
                     {
-                        await _schemaUpgradeRunner.ApplySchemaAsync(version: i, applyFullSchemaSnapshot: false, cancellationToken);
+                        await _schemaUpgradeRunner.ApplySchemaAsync(version: i, applyFullSchemaSnapshot: false, cancellationToken).ConfigureAwait(false);
                     }
                 }
 
-                await GetCurrentSchemaVersionAsync(cancellationToken);
+                await GetCurrentSchemaVersionAsync(cancellationToken).ConfigureAwait(false);
             }
         }
 
         private async Task GetCurrentSchemaVersionAsync(CancellationToken cancellationToken)
         {
-            using (var connection = await _sqlConnectionFactory.GetSqlConnectionAsync(cancellationToken: cancellationToken))
-            {
-                await connection.OpenAsync(cancellationToken);
+            using SqlConnection connection = await _sqlConnectionFactory.GetSqlConnectionAsync(cancellationToken: cancellationToken).ConfigureAwait(false);
+            await connection.OpenAsync(cancellationToken).ConfigureAwait(false);
 
-                string tableName = "dbo.SchemaVersion";
+            const string tableName = "dbo.SchemaVersion";
 
-                // Since now the status is made consistent as 'completed', we might have to check for 'complete' as well for the previous version's status
-                using (var selectCommand = connection.CreateCommand())
-                {
-                    selectCommand.CommandText = string.Format(
-                        "SELECT MAX(Version) FROM {0} " +
-                        "WHERE Status = 'complete' OR Status = 'completed'", tableName);
```

Comment:
The analyzer erroneously flagged this as a possible SQL injection because of the `string.Format`, but the `tableName` is actually constant